content/docs/addons-test-utils.md

```diff
@@ -223,7 +223,7 @@ scryRenderedDOMComponentsWithClass(
 )
 ```
 
-Finds all DOM elements of components in the rendered tree that are DOM components with the class name matching `className`.
+रेंडर किए गए ट्री में कौम्पोनॅन्ट के सभी DOM एलिमेंट्स को ढूँढता है जो DOM कौम्पोनॅन्ट हैं और जिनका नाम `className` से मितला है।
```

Comment:
मितला  -> spelling mistake

Reply:
fixed in b7c667cdd69b06a052dd48c32bb585e1